### PR TITLE
Skip testCreateSplitIndexToN on Windows

### LINKED ISSUE #92183: [CI] SplitIndexIT testCreateSplitIndexToN failing
**Build scan:**
https://gradle-enterprise.elastic.co/s/bvb5puxxlbaou/tests/:server:internalClusterTest/org.elasticsearch.action.admin.indices.create.SplitIndexIT/testCreateSplitIndexToN

**Reproduction line:**
```
gradlew ':server:internalClusterTest' --tests "org.elasticsearch.action.admin.indices.create.SplitIndexIT.testCreateSplitIndexToN" -Dtests.seed=24641F2A0356AEBD -Dtests.locale=ar-MA -Dtests.timezone=SystemV/EST5 -Druntime.java=19
```

**Applicable branches:**
8.6

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.action.admin.indices.create.SplitIndexIT&tests.test=testCreateSplitIndexToN

**Failure excerpt:**
```
java.lang.AssertionError: timed out waiting for green state

  at __randomizedtesting.SeedInfo.seed([24641F2A0356AEBD:92C8F9F3E9C1145A]:0)
  at org.junit.Assert.fail(Assert.java:88)
  at org.elasticsearch.test.ESIntegTestCase.ensureColor(ESIntegTestCase.java:966)
  at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:905)
  at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:894)
  at org.elasticsearch.action.admin.indices.create.SplitIndexIT.splitToN(SplitIndexIT.java:245)
  at org.elasticsearch.action.admin.indices.create.SplitIndexIT.testCreateSplitIndexToN(SplitIndexIT.java:78)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1589)

```

### PR DESCRIPTION
**Pull request by pxsalehi**
Please see https://github.com/elastic/elasticsearch/issues/92183#issuecomment-1419060755.

Closes https://github.com/elastic/elasticsearch/issues/92183

### CODE REVIEW

File: server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java [hunk 1]
```diff
@@ -72,15 +72,15 @@ protected boolean forbidPrivateIndexSettings() {
...
+        assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
+
         int[][] possibleShardSplits = new int[][] { { 2, 4, 8 }, { 3, 6, 12 }, { 1, 2, 4 } };
         int[] shardSplits = randomFrom(possibleShardSplits);
         splitToN(shardSplits[0], shardSplits[1], shardSplits[2]);
     }
 
     public void testSplitFromOneToN() {
-
         assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
```
Comment (DaveCTurner) [inline server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java:84]:
```suggestion
        assumeFalse("https://github.com/elastic/elasticsearch/issues/33857", Constants.WINDOWS);
```

File: server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java [hunk 2]
```diff
@@ -72,15 +72,15 @@ protected boolean forbidPrivateIndexSettings() {
         return false;
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/92183")
     public void testCreateSplitIndexToN() throws IOException {
+        assumeFalse("https://github.com/elastic/elasticsearch/issues/34080", Constants.WINDOWS);
```
Comment (DaveCTurner) [inline server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java:76]:
```suggestion
        assumeFalse("https://github.com/elastic/elasticsearch/issues/33857", Constants.WINDOWS);
```

**Review by DaveCTurner — APPROVED** [review]
Ah Windows, lovely Windows.

The approach is fine with me, but I think we should link #33857 rather than #34080 since the former issue is a better description of the problem.

**Comment by pxsalehi** [comment]
Thanks, David!

**Pull request merged**